src/eko/io/legacy.py

```diff
@@ -98,27 +101,29 @@ class PseudoOperator(DictLike):
 
     mu20: float
     mu2grid: npt.NDArray
-    rotations: Rotations
+    xgrid: XGrid
     configs: dict
 
     @classmethod
     def from_old(cls, old: RawCard):
         """Load from old metadata."""
         mu20 = float(old["q2_ref"])
-        mu2grid = np.array(old["Q2grid"])
+        mu2list = old.get("Q2grid")
+        if mu2list is None:
+            mu2list = old["mu2grid"]
+        mu2grid = np.array(mu2list)
 
         xgrid = XGrid(old["interpolation_xgrid"])
-        pids = old.get("pids", np.array(br.flavor_basis_pids))
 
-        rotations = Rotations(xgrid=xgrid, pids=pids)
+        rotations = Rotations(xgrid=xgrid)
```

Comment:
drop if no longer used (which I think is and will be the case)

Reply:
It was already changed in c5f0d1972c669797eae2d7dfb585d981b10e22d3
https://github.com/NNPDF/eko/commit/c5f0d1972c669797eae2d7dfb585d981b10e22d3#diff-3cb1f56e5e97e4f210cfa50e04bf1a2c30e49aeceb02c0f1e368738d4f4c4546L113-R112

Since the commit is from 10th of March, and your comment from 13th, I believe you just reviewed an old version...

Reply:
Done in 39bc095c4c0720bd5c3f3c9ad6a5cfac3254e40c